src/api/app/assets/javascripts/webui2/repositories.js

```diff
@@ -12,11 +12,76 @@ function setRepoCheckbox() { // jshint ignore:line
   $('.repocheckbox').click(function() {
     var id = $(this).attr('id');
     var $form;
-    if($(this).is(':checked')) {
+    if ($(this).is(':checked')) {
       $form = $('#' + id + '_create');
     } else {
       $form = $('#' + id + '_destroy');
     }
     $form.submit();
   });
 }
+
+function takeOverData(target, source, field) {
+  target.data(field, source.data(field));
```

Comment:
Hm, the function call is almost longer than the code it contains :thinking:

Reply:
true - it evolved a little - and then I didn't bother to fold it back in.

Reply:
But it avoids duplicating the field name.